test/scheduler/suite_test.go

```diff
@@ -0,0 +1,403 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package tests features a number of test suites that verify the behavior of the scheduler
+// and its related components.
+package tests
+
+import (
+	"context"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog/v2"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	clusterv1beta1 "go.goms.io/fleet/apis/cluster/v1beta1"
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	sched "go.goms.io/fleet/pkg/scheduler"
+	"go.goms.io/fleet/pkg/scheduler/clustereligibilitychecker"
+	fw "go.goms.io/fleet/pkg/scheduler/framework"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clusteraffinity"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clustereligibility"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/sameplacementaffinity"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/topologyspreadconstraints"
+	"go.goms.io/fleet/pkg/scheduler/queue"
+	crpwatcher "go.goms.io/fleet/pkg/scheduler/watchers/clusterresourceplacement"
+	pswatcher "go.goms.io/fleet/pkg/scheduler/watchers/clusterschedulingpolicysnapshot"
+	mcwatcher "go.goms.io/fleet/pkg/scheduler/watchers/membercluster"
```

Comment:
```suggestion
	"go.goms.io/fleet/pkg/scheduler"
	"go.goms.io/fleet/pkg/scheduler/clustereligibilitychecker"
	"go.goms.io/fleet/pkg/scheduler/framework"
	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clusteraffinity"
	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clustereligibility"
	"go.goms.io/fleet/pkg/scheduler/framework/plugins/sameplacementaffinity"
	"go.goms.io/fleet/pkg/scheduler/framework/plugins/topologyspreadconstraints"
	"go.goms.io/fleet/pkg/scheduler/queue"
	"go.goms.io/fleet/pkg/scheduler/watchers/clusterresourceplacement"
	"go.goms.io/fleet/pkg/scheduler/watchers/clusterschedulingpolicysnapshot"
	"go.goms.io/fleet/pkg/scheduler/watchers/membercluster"
```
missed this before, renaming is not necessary as there is no conflict.

& redundant empty line

Reply:
I don't particularly have a strong opinion on this one, though I do want to add that it's quite common in practice to rename K8s package for simplicity reasons, i.e., `controller-runtime` -> `ctrl`

Reply:
And the scheduler/framework rename is to avoid name collisions in the code.